Security/TwoFactor/Prodiver/U2F/U2FAuthenticator.php

```diff
@@ -26,7 +26,7 @@ public function __construct(RequestStack $requestStack)
         $scheme = $requestStack->getCurrentRequest()->getScheme();
         $host = $requestStack->getCurrentRequest()->getHost();
         $port = $requestStack->getCurrentRequest()->getPort();
-        $this->u2f = new \u2flib_server\U2F($scheme.'://'.$host.(('80' !== $port && '443' !== $port)?':'.$port:''));
+        $this->u2f = new \u2flib_server\U2F($scheme.'://'.$host.((80 !== $port && 443 !== $port)?':'.$port:''));
```

Comment:
This should be casted to int, because of this:
> can be a string if fetched from the server bag

https://github.com/symfony/symfony/blob/33d4bce1f9b931db0cbac0db3b7c6b272b16e7f9/src/Symfony/Component/HttpFoundation/Request.php#L997